Makefile

```diff
@@ -309,3 +309,10 @@ RAYROOT = $(shell $(GO_CMD) list -m -f "{{.Dir}}" github.com/ray-project/kuberay
 ray-operator-crd:
 	mkdir -p $(shell pwd)/dep-crds/ray-operator/
 	cp -f $(RAYROOT)/config/crd/bases/* $(shell pwd)/dep-crds/ray-operator/
+
+JOBSETROOT = $(shell $(GO_CMD) list -m -f "{{.Dir}}" sigs.k8s.io/jobset)
+.PHONY: jobset-operator-crd
+jobset-operator-crd:
+	mkdir -p $(shell pwd)/dep-crds/jobset-operator/
+	cp -f $(JOBSETROOT)/config/components/crd/bases/* $(shell pwd)/dep-crds/jobset-operator/
```

Comment:
```suggestion
	cp -f $(JOBSETROOT)/config/components/crd/bases/* $(PROJECT_DIR)/dep-crds/jobset-operator/
```

Reply:
this mix is all over the makefile, let's  create a followup for all of them #920